### PR TITLE
Updated Bug Issue template

### PR DESCRIPTION
**Pull request by smmr-dn**
## Changes

Changed **Bug Issue** template to require the link to a minirepo as well as current version information from users.

## Testing

Confirmed that the template turned out correctly.

## Docs

N/A

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (mayank99) [inline .github/ISSUE_TEMPLATE/bug_report.yml:26]: I think we could instead show a checkbox at the end?

[ ] I am using the latest version of iTwinUI.

Comment (smmr-dn) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Does this change okay? [commit](https://github.com/iTwin/iTwinUI/pull/2232/commits/4b0bc8baf4d81c60312a3079c921cdb0b083c023). Then users can specify their current version, if not the latest, in the last "Anything else?" text box.

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (mayank99) [inline .github/ISSUE_TEMPLATE/bug_report.yml:49]: I was thinking a single checkbox should be enough. If it is unchecked, it implies they are using a different version. Also we should include a link so that they know what the latest version is.

```suggestion
  - type: checkboxes
    attributes:
      label: Package Version
      options:
        - label: I'm using the [latest version](https://www.npmjs.com/package/@itwin/itwinui-react?activeTab=versions) of iTwinUI.
          required: false
    validations:
      required: false
```

Comment (smmr-dn) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: I'm not sure why the link markdown is not supported in `label` for checkbox `option`. I have looked around, but I haven't found a way to do as suggested. I have added a description to include the link in your suggestion but not sure this is okay.

Comment (mayank99) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Yeah I guess this is a limitation in Github, nothing we can do about it.

**Review by mayank99 — APPROVED** [review]
LGTM!

**Review by mayank99 — COMMENTED** [review]
Let's merge this, unless we're waiting for something.

(The "audit" failure doesn't block the PR)

**Pull request merged**
